src/store/initial-state.js

```diff
@@ -108,7 +152,16 @@ const getInitialState = (props = {}) => {
       disabled: { parameters: true, task: false, data: false },
     },
   });
-  const pipelineState = preparePipelineState(props.data, props.data !== 'json');
+
+  const expandAllPipelines =
+    nonPipelineState.display.expandAllPipelines ||
+    nonPipelineState.flags.expandAllPipelines;
+
+  const pipelineState = preparePipelineState(
+    props.data,
+    props.data !== 'json',
+    expandAllPipelines
+  );
```

Comment:
Possible to add a blank line before `return` statements? It's easier to read them I think.